src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -74,7 +75,10 @@ public IOperation GetOperation(SyntaxNode node, CancellationToken cancellationTo
             return this.GetOperationCore(node, cancellationToken);
         }
 
-        protected abstract IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken);
+        protected virtual IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
The use of  `abstract` here is not a breaking change.  This type has other `internal abstract` methods that make it not generally inheritable.  All published derived types are also `sealed`.